mrmustard/math/lattice/strategies/compactFock/inputValidation.py

```diff
@@ -18,7 +18,40 @@
 from mrmustard.math.lattice.strategies.compactFock.singleLeftoverMode_grad import (
     fock_representation_1leftoverMode_grad,
 )
-from thewalrus._hafnian import input_validation
+
+
+def input_validation(A, rtol=1e-05, atol=1e-08):
```

Comment:
I don't mind that it's a copy-paste from `thewalrus`, but maybe just leave a comment so people know it might drift from the original function 😄